libs/ett/auth/data-access-api/src/lib/oidc/auth-oidc-http.service.ts

```diff
@@ -49,12 +64,26 @@ export class AuthOidcHttp {
 
   login(returnUrl: string): Observable<AuthOidcLoginResponse> {
     return this.http.get<AuthOidcLoginResponse>(
-      `${environment.apiBase}/oidc/login`,
+      `${environment.apiBase}/auth/oidc/login`,
```

Comment:
praise: Nice! Ideally, we use the same paths locally as in a deployed environment as is the case here.